app/static/lang/de-DE.js

```diff
@@ -32,6 +32,7 @@ export default () => {
         'gb': 'English',
         'pl': 'Polski',
         'de': 'Deutsch',
+        'vnm': 'Tiếng Việt',
```

Comment:
These will generally default to the en-US file so they can be left alone assuming we use the localized versions there.